applications/asset_tracker_v2/doc/firmware_architecture.rst

```diff
@@ -24,18 +24,7 @@ It also shows the modules with thread and the modules without thread.
 Internal modules
 ****************
 
-The application comprises of the following modules:
-
-* :ref:`asset_tracker_v2_app_module`
-* :ref:`asset_tracker_v2_data_module`
-* :ref:`asset_tracker_v2_cloud_module`
-* :ref:`asset_tracker_v2_sensor_module`
-* :ref:`asset_tracker_v2_location_module`
-* :ref:`asset_tracker_v2_ui_module`
-* :ref:`asset_tracker_v2_util_module`
-* :ref:`asset_tracker_v2_debug_module`
-* :ref:`asset_tracker_v2_modem_module`
-
+The application comprises of a set of internal modules.
```

Comment:
**question:** why is this changed from "modules" to "internal modules"?

Reply:
@simensrostad can answer that, but I'd say do not change the meaning of the docs in this PR.

Reply:
Thanks for the nit, I like it. I was totally focused on the structural change and overlooked that. Fixed now.